tb_plugin/torch_tb_profiler/profiler/trace.py

```diff
@@ -21,6 +21,9 @@ class EventTypes(object):
     MEMSET = "Memset"
     PYTHON = "Python"
     MEMORY = "Memory"
+    CPU_OP = 'cpu_op'
+    GPU_MEMCPY = 'gpu_memcpy'
+    GPU_MEMSET = 'gpu_memset'
 
 Supported_EventTypes = [v for k, v in vars(EventTypes).items() if not k.startswith("_") and v != EventTypes.PROFILER_STEP]
```

Comment:
I am rethinking that the most simplest way to support the category changes would be adding a map from the underlying category in trace file to EventTypes here. Then in tb_plugin we will only need handle EventTypes check since it add a abstraction layer between libkineto and tb_plugin. Then we only need change the file itself.